sormas-api/src/main/java/de/symeda/sormas/api/user/UserRight.java

```diff
@@ -1007,6 +1007,14 @@
 	PERFORM_BULK_OPERATIONS(
 			ADMIN
 	),
+	PERFORM_BULK_OPERATIONS_EVENT(
+		ADMIN,
+		SURVEILLANCE_SUPERVISOR,
+		CONTACT_SUPERVISOR
+	),
+	PERFORM_BULK_OPERATIONS_EVENT_DELETE(
```

Comment:
This user right is not necessary; the decision whether or not the user sees the "Delete" bulk option should depend on whether or not they have the EVENT_DELETE user right